src/ArduinoJson.h

```diff
@@ -0,0 +1,8 @@
+// Copyright Benoit Blanchon 2014-2017
+// MIT License
+//
+// Arduino JSON library
+// https://github.com/bblanchon/ArduinoJson
+// If you like this project, please add a star!
+
+#include "../include/ArduinoJson.h"
```

Comment:
I'll try to move everything from `include/` to `src/` and see if other IDE agree.